accounts/store_test.go

```diff
@@ -71,6 +72,18 @@ func TestAccountStore(t *testing.T) {
 	)
 	require.NoError(t, err)
 
+	// Adjust the account balance by first adding 10000, and then deducting
+	// 5000.
```

Comment:
for review it's nicer if unit tests are in the same commit as the new addition :)